rest_framework_nested/runtests/settings.py

```diff
@@ -83,7 +85,7 @@
     # Don't forget to use absolute paths, not relative paths.
 )
 
-INSTALLED_APPS = (
+INSTALLED_APPS: tuple[str, ...] = (
```

Comment:
That surprised me, I confess. The tools are not able to infer this one?

Reply:
Mypy prefers to infer tuples as non-variadic, e.g. `tuple[str, str, str, str, str, str]` etc.

This causes problems since later lines add values to the tuple `INSTALLED_APPS += (`, changing its length and making it incompatible.

Changing it to a `list` is one possible alternative.

Reply:
Replaced them with list literal `[]` syntax